CHANGELOG.md

```diff
@@ -131,7 +131,7 @@
   * [FIX] Updated the Dockerfile so that the subdirectories are preserved in the Docker container.
 
 * **Metrics**
-  * [FIX] Added Service Monitor definition to permit metric collection from OpenSearch instance within individual SAS Viya deployments.
+  * [FIX] Added Service Monitor definition to permit metric collection from OpenSearch instance within individual SAS Viya platform deployments.
```

Comment:
The service monitor definitions will permit metric collection from any OpenSearch instance deployed within any flavor of SAS Viya, it is not limited to those instances deployed with SAS Viya platform.  Given that, the addition of "platform" seems inappropriate.